### PR TITLE
Add nginx in front of applications

### PR DESCRIPTION
**Pull request by adityapahuja**
This pull request adds nginx in front of all applications to act as a SSL termination and is used for encrypting communications between Application Load Balancer (ALB) and nginx over the network since they are not running on the same server or EC2 instance.

### CODE REVIEW

File: terraform/modules/hub/hub_config.tf [hunk 1]
```diff
@@ -18,14 +18,31 @@ module "config_ecs_asg" {
   logit_elasticsearch_url = "${var.logit_elasticsearch_url}"
 }
 
+locals {
+  config_location_blocks = <<-LOCATIONS
+  location = /healthcheck {
```
Comment (tlwr) [inline terraform/modules/hub/hub_config.tf]: can we remove the healthcheck and use /service-status for config?

Comment (tlwr) [reply terraform/modules/hub/hub_config.tf]: or rather /service status is used but this contains a healthcheck location for no reason

Comment (adityapahuja) [reply terraform/modules/hub/hub_config.tf]: Thanks for spotting. It looks like My rebase error has brought the old code back. I will remove it.

Comment (tlwr) [reply terraform/modules/hub/hub_config.tf]: :)

Comment (adityapahuja) [reply terraform/modules/hub/hub_config.tf]: I have resolved the rebase error. The old code is no longer present.

**Pull request merged**